### ISSUE TITLE
Add fixed site feature, to complement the timed-counts

### ISSUE THREAD

**Issue by kazlauskis** [NERC-CEH/irecord-butterflies-app#186]
Similar to how it works in the ButterflyCount app, this would allow setting up a fixed site and linking it to surveys.

**Comment by kazlauskis** [comment]
@DavidRoy should this be available to all surveys or just the 15min count?

**Comment by DavidRoy** [comment]
@kazlauskis just the 15min count (multi-species and single-species) to start with

**Comment by kazlauskis** [comment]
@DavidRoy In the ButterflyCount app, the fixed sites are directly linked to activities or groups, which means that users cannot select or save a site without first selecting an activity. I find this to be somewhat restrictive, as some users may simply want to save a site they revisit. What are your thoughts on this? Should we keep the sites linked to activities, or allow users to save sites independently?

**Comment by DavidRoy** [comment]
@kazlauskis I agree, we can allow sites to be saved without being linked to activities
